src/spdx_tools/spdx3/model/software/software_artifact.py

```diff
@@ -0,0 +1,26 @@
+#  SPDX-FileCopyrightText: 2023 spdx contributors
+#
+#  SPDX-License-Identifier: Apache-2.0
+#
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
```suggestion
```

Reply:
That's some stupid PyCharm bug, thanks for catching!